frontend/src/pages/org/components/new-browser-profile-dialog.ts

```diff
@@ -0,0 +1,182 @@
+import { state, property, queryAsync } from "lit/decorators.js";
```

Comment:
This file was copied pretty much wholesale from `browser-profile-list.ts`, with some changes to use `<btrix-dialog>` instead of `<sl-dialog>`.